### DISCUSSION TITLE
EasyVirtualDisplay 0.1

### DISCUSSION THREAD

**Discussion by KtzeAbyss** [KtzeAbyss/Easy-Virtual-Display#2] [General]
一键创建你的专属虚拟显示器
Effortlessly Generate Your Personalized Virtual Display

<hr /><em>This discussion was created from the release <a href='https://github.com/KtzeAbyss/Easy-Virtual-Display/releases/tag/'> EasyVirtualDisplay 0.1</a>.</em>

**Comment by adminYEE** [comment]
下个版本可以更新一下120hz吗，感觉120hz更契合手机一点。以及hdr的选项（感觉这个功能会复杂点）
